### PR TITLE
[GLUTEN-2024][CH] log build info

### PR DESCRIPTION
**Pull request by lwz9103**
## What changes were proposed in this pull request?

log build info on loading gluten plugin for clickhouse backend

(Fixes: \#2024)

## How was this patch tested?

manual tests

### CODE REVIEW

**Comment by lwz9103** [comment]
## Test Evidence

### clickhouse backend

File: gluten-core/src/main/scala/io/glutenproject/GlutenPlugin.scala [hunk 1]
```diff
@@ -77,13 +77,15 @@ private[glutenproject] class GlutenDriverPlugin extends DriverPlugin with Loggin
...
   private def showGlutenBuildInfo(): Unit = {
-    val veloxInfo = if (BackendsApiManager.veloxBackend) {
-      s"""
-        |Velox branch: $VELOX_BRANCH
-        |Velox revision: $VELOX_REVISION
-        |Velox revision time: $VELOX_REVISION_TIME""".stripMargin
-    } else {
-      ""
+    val backendInfo = BackendsApiManager.getBackendName.toLowerCase() match {
+      case "velox" => s"""
```
Comment (PHILO-HE) [inline gluten-core/src/main/scala/io/glutenproject/GlutenPlugin.scala:81]: Nit:
`BackendsApiManager.veloxBackend` / `BackendsApiManager.chBackend` can be directly used to recognize the current backend, which is added recently. Thanks!

Comment (lwz9103) [reply gluten-core/src/main/scala/io/glutenproject/GlutenPlugin.scala]: @PHILO-HE thanks, fixed.

**Review by baibaichen — APPROVED** [review]
LGTM

**Pull request merged**